# List of available Satellite constellations
# constellation_enum_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 GPS     = 0
	uint8 GLONASS = 1
	uint8 Galileo = 2
	uint8 SBAS    = 3
	uint8 QZSS    = 4
	uint8 BeiDou  = 5
	uint8 IRNSS   = 6
	uint8 Other   = 7
